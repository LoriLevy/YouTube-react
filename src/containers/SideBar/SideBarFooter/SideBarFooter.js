import React from 'react';
import './SideBarFooter.scss'

export function SideBarFooter() {
    return (
        <React.Fragment>
            <div className='footer-block'>
                <div>About</div>
                <div>Creator</div>
                <div>Developers +ViewTube</div>
                <div>Legal</div>
            </div>
            <div className='footer-block'>
                <div>Terms Privacy</div>
                <div>Policy & Safety</div>
                <div>Test new features</div>
            </div>
            <div className='footer-block'>
                <div>All prices include VAT</div>
            </div>
            <div className='footer-block'>
                <div>Â© ViewTube.com - A Youtube clone for my own educational purposes under fair use.</div>
            </div>
        </React.Fragment>
    );
}