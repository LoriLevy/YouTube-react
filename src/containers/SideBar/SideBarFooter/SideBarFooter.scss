div {
    color: #4c4d4f;
}

.footer-block {
    padding-bottom: 10px;
    padding-left: 16px;
    color: #918888;
}