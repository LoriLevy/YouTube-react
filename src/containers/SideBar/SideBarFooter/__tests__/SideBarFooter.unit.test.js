import React from "react";
import { shallow } from "enzyme";
import SideBarFooter from "../SideBarFooter";

describe("SideBarFooter", () => {
  test("it renders", () => {
    const wrapper = shallow(<SideBarFooter />);
    expect(wrapper).toMatchSnapshot();
  });
});
