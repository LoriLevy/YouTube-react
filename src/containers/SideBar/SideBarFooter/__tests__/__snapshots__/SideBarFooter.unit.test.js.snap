// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideBarFooter it renders 1`] = `
<Fragment>
  <div
    className="footer-block"
  >
    <div>
      About
    </div>
    <div>
      Creator
    </div>
    <div>
      Developers +ViewTube
    </div>
    <div>
      Legal
    </div>
  </div>
  <div
    className="footer-block"
  >
    <div>
      Terms Privacy
    </div>
    <div>
      Policy & Safety
    </div>
    <div>
      Test new features
    </div>
  </div>
  <div
    className="footer-block"
  >
    <div>
      All prices include VAT
    </div>
  </div>
  <div
    className="footer-block"
  >
    <div>
      Â© ViewTube.com - A Youtube clone for my own educational purposes under fair use.
    </div>
  </div>
</Fragment>
`;
